### ISSUE TITLE
cpplint does not understand C++11 attributes

### ISSUE THREAD

**Issue by mwoehlke-kitware** [google/styleguide#165]
A line like:

```
#define ABIEXPORT [[gnu::visibility("default")]]
```

...generates an incorrect warning:

```
file.h:#:  Extra space before [  [whitespace/braces]
```

**Comment by vapier** [comment]
public cpplint development in this repository has been halted (see #837).  please migrate to https://github.com/cpplint/cpplint now.
